src/Extensions/Abstractions/Activities/Processing/ActivityResultStrings.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+
+namespace Microsoft.Omex.Extensions.Abstractions.Activities.Processing
+{
+	/// <summary>
+	/// Class with activity result enum string values to avoid alocations
+	/// </summary>
+	public static class ActivityResultStrings
+	{
+		/// <summary>
+		/// Activity SystemError result string
+		/// </summary>
+		public static string SystemError { get; } = "SystemError";
+
+
+		/// <summary>
+		/// Activity ExpectedError result string
+		/// </summary>
+		public static string ExpectedError { get; } = "ExpectedError";
+
+
+		/// <summary>
+		/// Activity Success result string
+		/// </summary>
+		public static string Success { get; } = "Success";
+
+
+		/// <summary>
+		/// Activity Unknown result string
+		/// </summary>
+		public static string Unknown { get; } = "Unknown";
+
+
+		/// <summary>
+		/// Returns coresponding to enum string value with creating new string
+		/// </summary>
+		public static string ResultToString(TimedScopeResult result) =>
+			result switch
+			{
+				TimedScopeResult.SystemError => SystemError,
+				TimedScopeResult.ExpectedError => ExpectedError,
+				TimedScopeResult.Success => Success,
+#pragma warning disable CS0618 // need to handle all enum values
```

Comment:
`_` will throw an exception and intended for a case where a new value was added to enum, and `Unknown` is a valid value that we should not use. To be honest I'm not sure why it exists. But I see your point it might be easier to just throw in case of its usage

Reply:
Ok, whatever works for you in this case.

Reply:
It thinks you are right, it would be proper to throw an exception in case of Unknown

Reply:
Fixed, thank you